cd/python/pypi/pypi_publish.py

```diff
@@ -47,14 +48,14 @@ def post_wheel(path):
         print('In DEV account, skipping publish')
         print('Would have run: {}'.format(cmd))
         return 0
-    else:
+    elif any(test_version_mark in version for test_version_mark in ['a', 'b', 'dev']):
         print('Skipping publishing nightly builds to Pypi.')
         print('See https://github.com/pypa/pypi-support/issues/50 for details')
         return 0
 
+    else:
         # DO NOT PRINT CMD IN THIS BLOCK, includes password
-        p = subprocess.run(cmd.split(' '),
-                        stdout=subprocess.PIPE)
+        p = subprocess.run(cmd.split(' '), stdout=subprocess.PIPE)
```

Comment:
This is insecure because this will print the password in case of an exception. Passwords should always be passed using stdin

Reply:
twine doesn't have the option to pass credentials through stdin. I updated this part to pass credentials through environment variable in this process instead.